src/lib/components/molecules/column-chart/columnchart.stories.jsx

```diff
@@ -0,0 +1,50 @@
+import { ColumnChart } from '.'
+
+const meta = {
+  title: 'Molecules/ColumnChart',
+  component: ColumnChart,
+}
+
+export default meta
+
+export const Default = {
+  args: {
+    domain: [300, -100],
+    chartHeight: 400,
+    columnWidth: 20,
+    columns: [
+      {
+        value : 100,
+        id: "con",
+        color: "blue"
+      },
+      {
+        value : -100,
+        id: "lab",
+        color: "red"
+      },
+    ]
+  },
+  render: (args) => <ColumnChart {...args} />,
+}
+
+export const Councillors = {
+  args: {
+    domain: [300, -100],
+    chartHeight: 400,
+    columnWidth: 20,
+    columns: [
+      {
+        value : 100,
+        id: "con",
+        color: "blue"
+      },
+      {
+        value : -100,
+        id: "lab",
+        color: "red"
```

Comment:
Noticed that adding a color is something we have done two different ways - one directly with adding a fill, another where we use the abbreviation (or id here) to make use of the classes we have eg. `background-color--lab` etc. 

Eg in StackedBar Antonio has done it using abbreviations and using hex codes. Could be worth copying that and adding that option here:

https://github.com/guardian/interactive-component-library/blob/main/src/lib/components/particles/stacked-bar/stacked-bar.stories.jsx#L36

Reply:
thanks. I was wondering how to do that bit. I have also left the straightforward color declaration as an option, but it works fine with both declared; makes the component usable in contexts where there isn't already a defined colour scheme.